src/components/Loader/__stories__/Loader.stories.scss

```diff
@@ -1,7 +1,32 @@
 @import "../../../styles/themes.scss";
 
-.monday-storybook-loader {
-  width: 40px;
-  height: 40px;
-  position: relative !important;
+.monday-storybook-loader-on-primary-block {
```

Comment:
we usually using bem syntax for class names that not written with css modules.
this is not a blocker but please notice this in your next pr :)